samples/apps/copilot-chat-app/webapi/Hubs/ChatHub.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.Options;
+using SemanticKernel.Service.Config;
+using SemanticKernel.Service.Controllers;
+
+namespace SemanticKernel.Service.SignalR.Hubs;
+
+public class ChatHub : Hub
+{
+    private readonly ILogger<ChatHub> _logger;
+    public ChatHub(ILogger<ChatHub> logger)
```

Comment:
Please put a blank line between each class member (lines 12-13, 16-17) for readability's sake